### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by ctoth**
- **Fix typo in launcher locale**
- **Indicate that the game must be launched in order for the %appdata%\Factorio folder to appear**

### CODE REVIEW

**Comment by LevFendi** [comment]
Hello and thank you for your contribution! I will review and merge these corrections in a few hours or days. Please feel free to share any further comments on how the install instructions and etc. can be improved.

**Comment by ctoth** [comment]
The changes look more extensive than they are because I had to renumber the list to include the new item. I recommend we switch to the convention of using 1 for each number in the list to make future alteration easier, but that felt like a larger change than I should make for this one small fix.

**Comment by ahicks92** [comment]
If we assume the numbers shift a lot then you can't talk about them in a stable fashion anyway.  Should we bother keeping them numbered at all would be my question.

File: README.md [hunk 1]
```diff
@@ -65,18 +65,19 @@ To install a mod release, follow the instructions below:
...
-12. Try launching the game from Steam. This should now run the mod launcher instead, and say "Hello Factorio". If not setup correctly, the game might launch directly and you will hear music.
-13. Follow the mod launcher instructions while the game is configured. This may involve launching the game itself temporarily.
-14. The game is ready to play from the launcher when the main menu appears.
+3. Run the game at least once if you have not, and then exit it. This is necessary in order to create the folder for the next step.
+4. Navigate to your Factorio game data folder. This is inside a special Windows folder called "AppData". There are two ways to access the AppData folder, you either enter its short name using % signs, or you use the full path name. If you use the short name with the signs, the path is called `%AppData%/Factorio`. If you use the full path, the path depends on your windows user name and it is something like `C/Users/Your_User_Name_Here/AppData/Roaming/Factorio`.
```
Comment (ahicks92) [inline README.md:69]: This is missing a `:` after `C`.

File: README.md [hunk 2]
```diff
@@ -65,18 +65,19 @@ To install a mod release, follow the instructions below:
...
+5. Make sure you copied all of the contents in Step 2 and paste them into the Factorio game data folder you found in Step 3.
+6. The mod is now installed, but you still need to configure Steam and also the mod itself. Note that the game is both configured and played using "launcher.exe", which is necessary for the mod to read out the game. If you want, you can create a shortcut for this launcher.
+7. If you are a jaws screen reader user, you may want to copy the Factorio.jkm file into your JAWS settings folder, which is found in the AppData folder. An example file path is `C:\Users\Your_User_Name_Here\AppData\Roaming\Freedom Scientific\JAWS\2022\Settings\enu\`
+8. You can run the mod launcher directly or in administrator mode if needed. Running it the first time will generate a Windows security warning because you are running an unlicensed application.
```
Comment (ahicks92) [inline README.md:73]: Not unlicensed, unsigned.

File: README.md [hunk 3]
```diff
@@ -65,18 +65,19 @@ To install a mod release, follow the instructions below:
...
+10. Open your Steam Library and find the Factorio page.
+11. Find the Properties menu for Factorio. On the Steam Library, the Properties menu for a game can be opened by finding the game's name in the library list, and then right clicking it to open the context menu, and then selecting the last option called "Properties...". Alternatively, you can open the game's page in the Steam Library and open the "Manage" menu, which is denoted by a gear symbol that you can left click. From the "Manage" menu you can select the last option called "Properties..."
+12. When the Properties menu is open, you should be immediately at the "General" section. At the bottom of this section is the part called "Launch Options" with a text field that you can type in. Here, you need paste the mod launcher setup text string that you copied earlier.
```
Comment (ahicks92) [inline README.md:77]: should be need to

**Review by ahicks92 — COMMENTED** [review]
I can't tell which of these typos are and aren't new but marked them anyway.  Not sure what we can do about this problem in general that's easy; one way is to put up two commits one which inserts the new list items.  Yes that is annoying.  We may also want to wrap our md files, as this can make diffs easier with respect to knowing what is in the middmle of a paragraph.  VSCode's rewrap extension can handle it, but there might jus be a styluua for md.  Not inclined to spend the time on it right now.  Much bigger fish.

@LevFendi the info looks correct in the sense that it's the same as it was.

Curious what/why we have what seems to be non-in-game stuff in the Factorio localization file, but that's another topic even if (to me) a surprising one.

**Comment by LevFendi** [comment]
The typos are all mine 😁. Thanj you for the review notes, i will be fixing accordingly.

I just tried numbering all the points as "1" and indeed markdown arranges them by itself, that is really nice. We should keep the numbering because they are very useful when troubleshooting install problems.

As for the locale file here, it is for the FA launcher, which is already mostly localized using vanilla strings. This file covers the rest and its format will help future translators by having the same format as other locale files.

**Pull request merged**